src/components/reusable/Header.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+
+import { Routes } from '@/constants';
+import Logo from '@/assets/logo.svg';
+
+const Header = () => (
+  <Container fluid className="header px-1">
+    <Row className="mx-1">
+      <Link to={Routes.Intro} className="header__logo">
+        <img src={Logo}/>
+      </Link>
```

Comment:
@ashtonmeuser we should use <NavBar> for this instead. Menus will be styled better. But I will approve this for now